update_graph_objs.py

```diff
@@ -0,0 +1,271 @@
+from plotly.graph_objs import graph_objs_tools
+from plotly.graph_reference import ARRAYS, CLASSES
+
+FLAG = '# AUTO-GENERATED BELOW. DO NOT EDIT! See makefile.'
+
+
+def get_non_generated_file_lines():
+    """
+    Copy each line up to our special FLAG line and return.
+
+    :raises: (ValueError) If the FLAG isn't found.
+    :return: (list) The lines we're copying.
+
```

Comment:
I think the doc strings should not have the extra space between last line and the triple quotes """